easybuild/easyconfigs/p/pybind11/pybind11-2.10.4-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,23 @@
+name = 'pybind11'
+version = '2.10.4'
```

Comment:
It looks like you're not actually needing/using this version?

Reply:
pybind11 has been removed from PR.